screens/facility-profile.tsx

```diff
@@ -69,6 +73,83 @@ type FacilityProfileRouteProp = RouteProp<
   "FacilityProfile"
 >;
 
+const imageUrls = [
```

Comment:
Is this PR waiting on the image carousel PR to get merged?

Reply:
this PR already includes the image carousel PR-- will delete this